components/brave_wallet_ui/components/desktop/views/crypto/index.tsx

```diff
@@ -168,13 +170,34 @@ const CryptoView = (props: Props) => {
     onToggleAddModal()
   }
 
+  const selectAsset = (asset: TokenInfo) => {
+    if (asset) {
+      history.push(`${WalletRoutes.Portfolio}/${asset.symbol}`)
+    } else {
+      onSelectAsset(asset)
+      history.push(WalletRoutes.Portfolio)
+    }
+  }
+
+  const goBack = () => {
+    setSelectedAccount(undefined)
+    history.push(WalletRoutes.Accounts)
+    setHideNav(false)
+  }
+
+  const onSelectAccount = (account: WalletAccountType) => {
```

Comment:
`account` may be undefined, so type should reflect this I think.

Reply:
Done!